node/core/dispute-coordinator/src/real/spam_slots.rs

```diff
@@ -76,7 +81,10 @@ impl SpamSlots {
 		Self { slots, unconfirmed: unconfirmed_disputes }
 	}
 
-	/// Add an unconfirmed dispute if free slots are available.
+	/// Increase a "voting invalid" validator's spam slot.
+	///
+	/// This function should get called for any validator's invalidity vote for any not yet
+	/// confirmed dispute.
```

Comment:
Docs might indicate what the return value indicates.

Reply:
Thanks, I meant to add that :see_no_evil: